### PR TITLE
Add unit tests for load-balancer-and-ingress-service package 1.6.0

### PR DESCRIPTION
**Pull request by sbhatp**
Signed-off-by: Sharath Bhat <sharathb@vmware.com>

## What this PR does / why we need it
Add unit tests for load-balancer-and-ingress-service package 1.6.0

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
Add unit tests for load-balancer-and-ingress-service package 1.6.0
```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Comment by HanFa** [comment]
@sbhatp thanks for the PR! could you add a test to cover this ad-hoc logic https://github.com/vmware-tanzu/community-edition/pull/3255/files as well? that was the first motivation to do these unit test

File: addons/packages/load-balancer-and-ingress-service/1.6.1/test/go.mod [hunk 1]
```diff
@@ -0,0 +1,48 @@
+module github.com/vmware-tanzu/community-edition/addons/packages/vsphere-cpi/1.23.0-alpha.1/test
```
Comment (HanFa) [inline addons/packages/load-balancer-and-ingress-service/1.6.1/test/go.mod:1]: aha please change this line =)

File: addons/packages/load-balancer-and-ingress-service/1.6.1/test/load_balancer_and_ingress_service_test.go [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+var _ = Describe("LoadBalancer And Ingress Service Ytt Templates", func() {
+	var (
+		filePaths    []string
+		values       string
+		output       string
+		yttRenderErr error
+
+		configDir = filepath.Join(repo.RootDir(), "addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config")
+
+		// vsphere-cpi mode
```
Comment (HanFa) [inline addons/packages/load-balancer-and-ingress-service/1.6.1/test/load_balancer_and_ingress_service_test.go:28]: plz correct it

File: addons/packages/load-balancer-and-ingress-service/1.6.1/test/go.mod (hunk 1 above)
Comment (sbhatp) [reply addons/packages/load-balancer-and-ingress-service/1.6.1/test/go.mod:1]: good catch

**Comment by sbhatp** [comment]
> @sbhatp thanks for the PR! could you add a test to cover this ad-hoc logic https://github.com/vmware-tanzu/community-edition/pull/3255/files as well? that was the first motivation to do these unit test

Thanks, done

**Review by HanFa — APPROVED** [review]
/lgtm
/approve

**Pull request merged**